Common subdirectories: xtarot-1.0.3/cards and xtarot-1.0.3_patched/cards
Common subdirectories: xtarot-1.0.3/tdeck and xtarot-1.0.3_patched/tdeck
diff -u xtarot-1.0.3/xtarot.h xtarot-1.0.3_patched/xtarot.h
--- xtarot-1.0.3/xtarot.h	2001-01-02 00:43:34.000000000 +0100
+++ xtarot-1.0.3_patched/xtarot.h	2007-11-08 06:17:18.000000000 +0100
@@ -19,10 +19,10 @@
 
 #ifndef _XTAROT_H_
 #define _XTAROT_H_
-#define FONT1 "-b&h-lucidatypewriter-*-*-*-*-*-*-*-*-*-100-*-*" 
-#define FONT2 "-b&h-lucidatypewriter-*-*-*-*-*-*-*-*-*-75-*-*"
-#define FONT3 "-*-chancery l-*-*-*-*-25-*-*-*-*-*-*-*"
-#define FONT4 "-*-chancery l-*-*-*-*-18-*-*-*-*-*-*-*"
+#define FONT1 "-*-*-*-*-*-*-*-*-*-*-*-100-*-*" 
+#define FONT2 "-*-*-*-*-*-*-*-*-*-*-*-75-*-*"
+#define FONT3 "-*-*-*-*-*-*-25-*-*-*-*-*-*-*"
+#define FONT4 "-*-*-*-*-*-*-18-*-*-*-*-*-*-*"
 
 Display *disp;
 ImlibData *id;
